app/views/user_mailer/daily_digest.html.erb

```diff
@@ -46,10 +46,10 @@
   <% end %>
 
   <% if @updates[:community_updates].present? %>
-    <% question_count = @updates[:community_updates].values.map { |v| v[:questions].length }.sum %>
+    <% topics_count = @updates[:community_updates].values.map { |v| v[:topics].length }.sum %>
```

Comment:
`UserMailerPreview` hasn't been updated to match this change. i.e., `:topics` vs `:questions`.